### PR TITLE
[Discuss] New issue forms 

### PR DESCRIPTION
**Pull request by bmorelli25**
Hello, @elastic/obs-docs 👋.

After our discussion today, I wanted to play around with issue forms and try to figure out a way to add target version labels to issues automatically. Unfortunately, the only way to add a version label was to create different forms for each release. But maybe this actually works for us? Here's what it would look like when someone wants to create an issue in this repo:

Screen Shot 2022-10-06 at 1 25 58 PM

Users select a target release version and are presented with an issue form for that version. In this example, the user selected 8.6, which would automatically add the `8.6-request` label to the issue. In addition, all fields are required—forcing whoever creates the issue to give us as many juicy details as possible.

Screen Shot 2022-10-06 at 1 24 15 PM

### CODE REVIEW

**Review by colleenmcginnis — COMMENTED** [review]
I love this!! One small q below.

File: .github/ISSUE_TEMPLATE/8.6-request.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Please link to as many related resources as possible. Think about issues, PRs, demos, Slack threads, etc.
+      placeholder: How can the docs team learn more about this feature?
+    validations:
+      required: true
+  - type: checkboxes
```
[the remark this answers is not in the document]
Comment (bmorelli25) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: I initially used a `dropdown` but changed it for two reasons:

* No markdown support (this isn't a good reason)
* The dropdown boxes are really small. They wrap at like 40 chars, so it wasn't the easiest to read.

[the remark this answers is not in the document]
Comment (colleenmcginnis) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: Grr. Feature request for GitHub (if you're listening 👋 ): Radio buttons!

[the remark this answers is not in the document]
Comment (bmorelli25) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: 💯

**Comment by dedemorton** [comment]
I like it! We might want to tweak the language a bit, though.

I might be too literal, but "Request an update..." feels to me like it applies to existing content, not new stuff. Can we take this opportunity to broaden the description? How about something like, "Request a documentation change or enhancement..."

Also the phrase "....isn't tied to an Elastic Stack release" might be misleading. For example, what if someone wants us to update the release notes for an earlier version because something is missing? I think someone reading this might be confused about what to do. Maybe say something like: "Request a documentation change or enhancement for any supported release...."

Just spitballing....

File: .github/ISSUE_TEMPLATE/8.6-request.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    id: collaboration
+    attributes:
+      label: Collaboration
+      description: Choose the expected collaboration model
+      options:
+        - label: The **documentation** team will proide the initial content.
```
Comment (ollyhowell) [inline .github/ISSUE_TEMPLATE/8.6-request.yaml:32]:
```suggestion
        - label: The **documentation** team will investigate the issue and create the initial content.
```

Comment (ollyhowell) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:32]: I was going to just suggest a simple typo fix but I wonder if it's worth rephrasing to emphasis more precisely what we would do?

File: .github/ISSUE_TEMPLATE/8.6-request.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+name: 8.6 Documentation request
+description: Request a documentation change or enhancement for Elastic Stack version 8.6.
+title: "[REQUEST]: "
+labels: ["8.6-request", "Team:Docs"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this request! This form will create an issue that the obs-docs team will triage and prioritize. Filling out this form does not guarantee that your issue will be prioritized for the selected iteration.
```
Comment (ollyhowell) [inline .github/ISSUE_TEMPLATE/8.6-request.yaml:9]:
```suggestion
        Thanks for taking the time to fill out this request! This form will create an issue that the obs-docs team will triage and prioritize. Filling out this form does not guarantee that your issue will be prioritized for the selected iteration but completing this issue as early and as comprehensively as possible will help us understand and plan the work better!
```

File: .github/ISSUE_TEMPLATE/8.7-request.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      label: Description
+      description: Describe what needs to be documented.
+      placeholder: I need...
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Resources
+      description: Please link to as many related resources as possible. Think about issues, PRs, demos, Slack threads, etc.
```
Comment (ollyhowell) [inline .github/ISSUE_TEMPLATE/8.7-request.yaml:22]:
```suggestion
      description: Please link to as many related resources as possible. Think about issues, PRs, demos, Slack threads, existing or related docs pages, etc.
```

File: .github/ISSUE_TEMPLATE/8.6-request.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  - type: checkboxes
+    id: collaboration
+    attributes:
+      label: Collaboration
+      description: Choose the expected collaboration model
+      options:
+        - label: The **documentation** team will proide the initial content.
+        - label: The **product** team will provide the initial content.
+        - label: TBD. The docs and product team will work together to determine the best path forward.
+        - label: Other (please describe above).
```
Comment (ollyhowell) [inline .github/ISSUE_TEMPLATE/8.6-request.yaml:35]:
```suggestion
        - label: Other (please describe below within **Point of contact** section).
```

File: .github/ISSUE_TEMPLATE/8.6-request.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ollyhowell) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: That's a shame that dropdowns are not well designed - would be nice to see a working example to see if it's worth cutting our text to fit dropdowns...think we can survive without markdown tho (was it just for bold etc.?).

**Comment by bmorelli25** [comment]
We might also consider prepending these three forms with `[Internal]` and adding a fourth form for external users. External (community) users don't need to fill out most of this information—I think we're more interested in links to pages that need to be updated and an explanation of why they think the docs are out of date or wrong.

That would make the issue options something like this:

* [Community] Documentation request
* [Internal] 8.6 Documentation request
* [Internal] 8.7 Documentation request
* [Internal] General documentation request

**Comment by bmorelli25** [comment]
With community form:

Screen Shot 2022-10-11 at 12 35 05 PM

File: .github/ISSUE_TEMPLATE/8.6-request.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmorelli25) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: Okay, I guess it's not _that_ bad. What do you think? Swap to dropdown?

Screen Shot 2022-10-12 at 8 09 57 AM

[the remark this answers is not in the document]
Comment (bmorelli25) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml:26]: After selection

Screen Shot 2022-10-12 at 8 11 30 AM

[the remark this answers is not in the document]
Comment (bmorelli25) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml]: I see one thumbs up already. Changed in [`63fb4f2`](https://github.com/elastic/observability-docs/pull/2231/commits/63fb4f2411ffaba5a8d54f5fc3cfe972cfe02f10).

[the remark this answers is not in the document]
Comment (ollyhowell) [reply .github/ISSUE_TEMPLATE/8.6-request.yaml]: We can always reduce the text further if necessary but your screenshot looks ok to me

**Comment by dedemorton** [comment]
I like what you've done here! How are we planning to track ongoing maintenance of these templates. When we get the notification to create the new branch, I assume will also update the templates?

**Comment by bmorelli25** [comment]
> How are we planning to track ongoing maintenance of these templates? When we get the notification to create the new branch, I assume will also update the templates?

Maintenance is a concern as the updating process will be manual. Branch creation could be a good time or just before a minor release.

File: .github/ISSUE_TEMPLATE/8.6-request.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+name: "[Internal] 8.6 Documentation request"
+description: Request a documentation change or enhancement for Elastic Stack version 8.6 (Elastic employees).
```
Comment (bmorelli25) [inline .github/ISSUE_TEMPLATE/8.6-request.yaml:2]:
```suggestion
description: Request a documentation change or enhancement for the 8.6 release (Elastic employees).
```

File: .github/ISSUE_TEMPLATE/8.7-request.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+name: "[Internal] 8.7 Documentation request"
+description: Request a documentation change or enhancement for Elastic Stack version 8.7 (Elastic employees).
```
Comment (bmorelli25) [inline .github/ISSUE_TEMPLATE/8.7-request.yaml:2]:
```suggestion
description: Request a documentation change or enhancement for the 8.7 release (Elastic employees).
```

File: .github/ISSUE_TEMPLATE/0-community-request.yaml [hunk 8]
```diff
@@ -0,0 +1,31 @@
+name: "[Community] Documentation request"
+description: Request a documentation change or enhancement.
+title: "[REQUEST]: "
+labels: ["request", "Team:Docs", "community"]
```
Comment (bmorelli25) [inline .github/ISSUE_TEMPLATE/0-community-request.yaml:4]:
```suggestion
labels: ["request", "Team:Docs", "💝 community"]
```

**Review by dedemorton — APPROVED** [review]
LGTM!  That's Looks GREAT to me. Thanks so much for doing this.

**Pull request merged**